rotkehlchen/db/upgrade_manager.py

```diff
@@ -133,6 +134,14 @@ def _perform_single_upgrade(self, upgrade: UpgradeRecord) -> None:
         if current_version != upgrade.from_version:
             return
         to_version = upgrade.from_version + 1
+        self.db.msg_aggregator.add_message(
+            message_type=WSMessageType.LOGIN_STATUS,
+            data={
```

Comment:
I see. So you just add a message just before a single upgrade is starting.

This is not really useful.
Why not give more information? Especially since 99% of the users do a single upgrade.

Add progress. Add progress inside each upgrade. We can have progress messages before and after a certain function is starting. This way we can give a % progress bar for each individual upgrade.

Also add a message once an upgrade has finished.